backend/src/main/java/ai/giskard/service/UserService.java

```diff
@@ -198,6 +198,16 @@ public void deleteUser(String login) {
             });
     }
 
+    public void restoreUser(String login) {
+        userRepository
+            .findOneByLogin(login)
+            .ifPresent(user -> {
+                user.setEnabled(true);
```

Comment:
since the flag is called `enabled` let's keep this term everywhere for consistency. So let's use "enable" in the method name and the log message. Log level can also be `info` here